src/db.ts

```diff
@@ -0,0 +1,11 @@
+import { Pool } from "pg";
+
+const pool = new Pool({
+  user: "juggler",
+  password: "juggleCat",
```

Comment:
meep shouldn't have this here

Reply:
Temp for us to get things setup- will replace w/ .env variables once we get going